coding-standard/README.md

```diff
@@ -0,0 +1,30 @@
+# Magento 2 Coding Standard Action
+
+A Github Action that runs the Magento Coding Standard.
+
+## Inputs
+
+See the [action.yml](./action.yml)
+
+## Usage
+
+```yml
+name: Coding Standard
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  coding-standard:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: graycoreio/github-actions-magento2/coding-standard@main
```

Comment:
These instructions need to be updated as soon as this has been moved to the infrastructure repo.